lib/active_record/sql_analyzer/monkeypatches/query.rb

```diff
@@ -9,7 +10,7 @@ def execute(sql, *args)
           safe_sql = nil
 
           SqlAnalyzer.config[:analyzers].each do |analyzer|
-            if SqlAnalyzer.config[:should_log_sample_proc].call(analyzer[:name])
+            if _square_private_should_sample_query(analyzer[:name])
```

Comment:
Updated to different namespace. I want a specific "fake namespace" to avoid clashing with (for example) someone else's monkeypatch of the same object.